torch/_torch_docs.py

```diff
@@ -2220,10 +2218,63 @@ def merge_dicts(*dicts):
 
 Example::
 
-    >>> torch.conj(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
+    >>> torch.conj_physical(torch.tensor([-1 + 1j, -2 + 2j, 3 - 3j]))
     tensor([-1 - 1j, -2 - 2j, 3 + 3j])
 """.format(**common_args))
 
+add_docstr(torch.conj,
+           r"""
+conj(input) -> Tensor
+
+Returns a view of :attr:`input` with the conjugate bit set to the negated value of :attr:`input`'s
+conjugate bit. If :attr:`input` has a non-complex dtype, this function just returns :attr:`input`.
+
+.. note::
+    :func:`torch.conj` performs a lazy conjugation, but the actual conjugated tensor can be materialized
```

Comment:
"performs a lazy conjugation" -> "can be thought of as performing a "lazy" conjugation"?

What about ending the sentence after ""lazy" conjugation" and including an example of the intended use?

Should `resolve_conj()` and `is_conj()` be in a see also section?